### ISSUE TITLE
Unique HTML IDs for article headers

### ISSUE THREAD

**Issue by QmwJlHuSg9pa** [good/openweb#66]
Article headers (e.g. "RSS Readers & Discovery") are missing unique HTML IDs. These would be handy for accessibility purposes, and for directly hyperlinking to particular sections of the page.

**Comment by splorp** [comment]
Good point. I’ll get on that.

**Comment by good** [comment]
Great idea! Just pushed that out.

**Comment by splorp** [comment]
Woo!
